src/layouts/PageLayout/PageLayout.module.scss

```diff
@@ -40,6 +40,10 @@
 
       .bk-page-layout__header__scope-switcher {
         @include typography.h6;
+        
+        > div {
+          border: none;
+        }
```

Comment:
Suggestion: you can move this to the `baklava.overrides` layer to ensure it overrides the `Select` component (which is in the same `baklava.components` layer):

```css
@layer baklava.components {
  ...
}
@layer baklava.overrides {
  .bk-page-layout__header__scope-switcher > div {
    border: none;
  }
}
```